platforms/documentation/docs/src/docs/userguide/reference/third_party_integration.adoc

```diff
@@ -15,139 +15,4 @@
 [[third_party_integration]]
 = Gradle & Third-party Tools
 
-Gradle can be integrated with many different third-party tools such as IDEs and continuous integration platforms. Here we look at some of the more common ones as well as how to integrate your own tool with Gradle.
-
-[[ides]]
-== IDEs
```

Comment:
Are you moving it to the Cookbook? AFAICT we do not have a replacement content
As of now, I suggest incorporating the difference to https://docs.gradle.org/current/userguide/gradle_ides.html or moving the pages to the Cookbook and putting references